types/heic-convert/index.d.ts

```diff
@@ -0,0 +1,30 @@
+// Type definitions for heic-convert 1.2
+// Project: https://github.com/catdad-experiments/heic-convert
+// Definitions by: Tregan <https://github.com/treemmett>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface ConversionOptions {
+    /**
+     * the HEIC file buffer
+     */
+    buffer: ArrayBufferLike;
+    /**
+     * output format
+     */
+    format: 'JPEG' | 'PNG';
+    /**
+     * the JPEG compression quality, between 0 and 1
+     */
+    quality?: number;
+}
+
+interface Convertible {
+    convert(): Promise<ArrayBuffer>;
+}
+
+declare function convert(image: ConversionOptions): Promise<ArrayBuffer>;
```

Comment:
```suggestion
/** @async */
declare function convert(image: ConversionOptions): Promise<ArrayBuffer>;
```
https://jsdoc.app/tags-async.html
(TS declaration file does not support `async` keyword)